bliss/models/encoder.py

```diff
@@ -177,61 +175,45 @@ def __init__(
 
         """
         super(ImageEncoder, self).__init__()
+
         # image parameters
         self.n_bands = n_bands
-        self.background_pad_value = background_pad_value
 
         # padding
         self.tile_slen = tile_slen
         self.ptile_slen = ptile_slen
-        self.edge_padding = (ptile_slen - tile_slen) / 2
-        assert self.edge_padding % 1 == 0, "amount of padding should be an integer"
-        self.edge_padding = int(self.edge_padding)
-        self.pad_border_w_constant = pad_border_w_constant
+        border_padding = (ptile_slen - tile_slen) / 2
```

Comment:
thanks! this variable name is now consistent :) 

should have thought of this before...